.github/README.md

```diff
@@ -1,164 +1,74 @@
 
 ## Philosophy
 
-Our main goal is to create a playable game server, offering a fully working game experience.
+Our main goal is to create a playable game server, offering a fully working in-game experience.
 
 Here are the main points we focus on:
 
 * Stability
-* Ease of use / Practicability
-* Playability (in-game content)
-* Customization
-* Community-driven software (check our discord)
-
-We also welcome new users (even non-English speaking users!) and help them learn/improve their skills (C++, SQL, Git, software collaboration, tutoring/wiki, etc...).
-
-Unlike other projects which focus more on the developer's side of things, we want users to be able to run their server with as few troubles as possible. All of our contributors run their private servers (local or public).
+  * We make sure all changes pass the CIs before being merged into the master branch.
 
-That's why AzerothCore is easier to use, to maintain, to understand, to develop on, and to customize to suit your needs, than other emulators.
-
-In short, we focus on the **user experience (UX)**, whether it be the **player's experience**, the **developer's experience**, or the **administrator's experience**.
-
-
-## How to Thank us
-
-Being an open-source project, we rely on volunteers to pursue development. Here are ways to help us if you use AzerothCore:
-
-### Github Star
-
-Click on the "star this repository" button to help us gain more visibility on Github!
-
-### By contributing
-
-Check the **CONTRIBUTING** section below.
-
-### Financially :moneybag:
-
-You can support the AzerothCore by [donating](https://www.paypal.com/donate/?hosted_button_id=L69ANPSR8BJDU).
-The money will be used to pay freelance developers for more open-source fixes.
-
-### Advertising
-
-By talking about us on different platforms or to people who would like to get involved.
-
-
-## Contributing
-
-AzerothCore is a learning project, and there are lots of different ways to contribute to the project:
-
-* By [testing our fixes](http://www.azerothcore.org/wiki/How-to-test-a-PR) (we can teach you how to correctly use Git to help us but that will also help you out tremendously)
-* By developing directly to the core or the modules
-* By reporting bugs within the project
-* By [creating new modules](http://www.azerothcore.org/wiki/Create-a-Module)
-* By improving our wiki
-* By providing direct support to our community (on Discord, StackOverflow or specialized forums)
-* By making extra content (video tutorial for example)
-* By putting bounties on issues
+* Blizzlike content
+  * We strive to make all in-game content to be blizzlike. Therefore we have a high standard for fixes being made.
 
+* Customization
+  * It should be easy to customize your experience using [modules](#modules).
```

Comment:
This seems like a weird way to phrase it, but I don't know what a better way would be. Just seems oddly hopeful.

Reply:
perhaps we can change it to `It is easy to... `

Reply:
> perhaps we can change it to `It is easy to... `

Works for me. 👍

Reply:
```suggestion
  * It is easy to customize your experience using [modules](#modules).
```